hawkbit-rest/hawkbit-mgmt-resource/src/test/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtDistributionSetTypeResourceTest.java

```diff
@@ -397,6 +397,7 @@ public void removeOptionalModuleToDistributionSetType() throws Exception {
                 appType.getId()).contentType(MediaType.APPLICATION_JSON)).andDo(MockMvcResultPrinter.print())
                 .andExpect(status().isOk());
 
+        Thread.sleep(1000);
```

Comment:
try to avoid using `Thread.sleep`, if absolutely necessary then maybe consider `Awaitlity`

Reply:
removed